src/assets/sass/protocol/components/_button.scss

```diff
@@ -4,35 +4,100 @@
 
 @import '../includes/lib';
 
+$field-focus-ring: 0 0 0 2px rgba($color-blue-40, .5);
```

Comment:
Should we use an opaque light blue instead of alpha? The transparency isn't very visible on a dark background but maybe that's fine since there's still the border. Or we could maybe make it a little more opaque just in the dark theme.

Reply:
I was following the specs Wendy provided but I agree it's hard to see. I'd be happy to just put it on a white matte (making it SUPER bright). But will defer to @justinkropp if he has an opinion

Reply:
Good question. I think the border would suffice here, but perhaps the alpha could be bumped up to `.7` or `.8` and see if that helps.